api/newsletters/template.py

```diff
@@ -0,0 +1,401 @@
+BTN_PRIMARY = "#3498db"
+BTN_SECONDARY = "#43ff8c"
+BTN_TERTIARY = "#db344d"
+
+STYLE = """ <style>
+
+      img {
+        border: none;
+        -ms-interpolation-mode: bicubic;
+        max-width: 85%;
+      }
+
+      body {
```

Comment:
@leonkoech - no need to fix this, but see how ugly this is? How did you test what the template looked like? Adding HTML in a string like this is a no-no because it's challenging to troubleshoot and update.  What you want to aim for is a template file for your HTML (e.g. Jinja https://jinja.palletsprojects.com/en/3.1.x/) where that is used, and variables are rendered on top of it.

Reply:
I was super confused on how to do this. Documented it as an issue [here](https://github.com/opportunity-hack/frontend-ohack.dev/issues/9#issuecomment-1406035705) 
I will absolutely consider using something that would make it easier to trouble shoot. 

Another challenge I noticed was adding variables to the style string to make it easier to change colors for primary, secondary, and tertiary buttons. We can discuss more on this @gregv